src/ui/src/app/modules/user/pages/login-register/login-register.component.ts

```diff
@@ -0,0 +1,62 @@
+import { Component, OnInit } from '@angular/core';
+import {FormBuilder, FormGroup, Validators} from "@angular/forms";
+import {AuthenticationService} from "../../../../core/services/authentication/authentication.service";
+
+@Component({
+  selector: 'app-login-register',
+  templateUrl: './login-register.component.html',
+  styleUrls: ['./login-register.component.css']
+})
+export class LoginRegisterComponent implements OnInit {
+
+  hide = true;
+  repeatedHide = true;
+  access: FormGroup;
+  message: string;
+
+  constructor(public formBuilder: FormBuilder, public authService:AuthenticationService) {
+  }
+
+  ngOnInit(): void {
+    this.access = this.formBuilder.group({
+      username: ["", Validators.required],
+      email: ["", Validators.required],
+      password: ["", Validators.required],
+      repeatPassword: ["", Validators.required],
+      login: true,
+    })
+  }
+
+  login(){
+    if(this.access.get("password").value !== this.access.get("repeatPassword").value &&
+      !this.access.get("login").value){
+      this.message = "enter the same password"
+    }else{
+      this.message = "";
+    }
+    if(this.access.get("login").value){
+      this.authService.login(
```

Comment:
subscribe to observable to execute the login.   (Remember to unsubscripe in the OnDelete Lifecycle hook

Reply:
Unsubscribe from subscription in ngOnDelete handler. See StatsComponent for more information. Memory leak alert